habitat_baselines/config/imagenav/ddppo_imagenav_gibson.yaml

```diff
@@ -43,10 +45,15 @@ RL:
     use_normalized_advantage: False
 
     hidden_size: 512
+
+    # Use double buffered sampling, typically helps
```

Comment:
Valuable comment, let's move to the doc string then it will be part of the documentation.

Reply:
Do we have docs for the config?

Reply:
No, we don't. But at leas have this repeated where the parameter is used would be great.